### PR TITLE
Update FlightCore guide, mention Repair window

### PR DESCRIPTION
**Pull request by itscynxx**
Updates the guide to include new features for FlightCore, and change asterisks to underscores to better fit formatting

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
Doesn't this need the changed images from #144 ?

**Comment by itscynxx** [comment]
yes, it relies on having the images merged just to give a visual to what the screen they're looking for is

i can edit this if needed

**Comment by GeckoEidechse** [comment]
Probably best to just add the images to this PR then and close the other one ^^

File: docs/installing-northstar/northstar-installers/flightcore-guide.md [hunk 1]
```diff
@@ -50,4 +50,10 @@ The *Mods* tab of Flightcore can display either mods installed locally, or a bro
...
+The _Settings_ tab is the fourth and final tab of FlightCore. This tab allows you to view and choose your Titanfall2 directory (default directories can be found in the [Default Directories](../troubleshooting.md#game-location) section), select how many mods you want to see per page on the online browser in FlightCore, the version of FlightCore you're running, a toggle to enable testing release channels, and an option to see a tab for quick access to options designed to help you troubleshoot NorthStar, labeled _Repair_.
+
+![FlightCore's Repair tab](../../images/flightcore-repair.png)
+
+This tab is very useful to both you and anyone that might be trying to help you, with both general Northstar options and FlightCore specific options. The `_"Disable all but core mods"_` button allows you to quickly disable all mods that are installed after installing Northstar, allowing you to quickly see if a mod you have installed is causing issues. `_"Force reinstall Northstar"_` does what the name implies, completely reinstalling Northstar in the event of something going wrong with your Northstar files.
```
Comment (GeckoEidechse) [inline docs/installing-northstar/northstar-installers/flightcore-guide.md]: Sticking formatted text (i.e. the italics) into a code section renders it like this

> `_Hello World_`

in standard markdown (unless GitBook does something different).

I don't think that's what you intended it to look like?

I'd just drop the backticks (`` ` ``). Quotes + italics should be enough ^^

**Review by GeckoEidechse — APPROVED** [review]
Aight looks good now. Merging ^^

**Pull request merged**